core/meterer/paymentlogic/paymentlogic.go

```diff
@@ -0,0 +1,166 @@
+package paymentlogic
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"math/big"
+	"slices"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/core"
+)
+
+// GetReservationBinLimit returns the bin limit for a given reservation
+// Note: This is called per-quorum since reservation is for a single quorum.
+func GetReservationBinLimit(reservation *core.ReservedPayment, reservationWindow uint64) uint64 {
+	return GetBinLimit(reservation.SymbolsPerSecond, reservationWindow)
```

Comment:
rename to `reservationWindowSeconds` everywhere?

Reply:
Also do we really need both GetReservationBinLimit and GetBinLimit? Could we only use one?